source/MapPanel.cpp

```diff
@@ -1059,32 +1070,35 @@ void MapPanel::DrawWormholes()
 			continue;
 
 		for(auto &&link : it.second.Links())
-			if(p.IsInSystem(link.first)
-					&& player.HasVisited(*link.first) && player.HasVisited(*link.second))
-				arrowsToDraw.emplace(link.first, link.second);
+			if(p.IsInSystem(link.first) && player.HasVisited(*link.first) && player.HasVisited(*link.second))
```

Comment:
one character too long
```suggestion
			if(p.IsInSystem(link.first) && player.HasVisited(*link.first)
					&& player.HasVisited(*link.second))
```